### PR TITLE
Add aggregatable source and aggregatable trigger

### PR DESCRIPTION
**Pull request by linnan-github**
Introduced new structs for aggregatable trigger data and aggregatable trigger, added aggregatable source and aggregatable trigger to attribution source and attribution trigger.

Also introduced new struct for aggregatable histogram.

***
<a href="https://pr-preview.s3.amazonaws.com/WICG/conversion-measurement-api/pull/400.html" title="Last updated on May 9, 2022, 5:33 PM UTC (b3418ca)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/conversion-measurement-api/400/a30026e...b3418ca.html" title="Last updated on May 9, 2022, 5:33 PM UTC (b3418ca)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -352,6 +352,38 @@ An attribution source is a [=struct=] with the following items:
...
+An attribution aggregatable trigger data is a [=struct=] with the following items:
+
+<dl dfn-for="attribution aggregatable trigger data">
+: key piece
+:: A non-negative 128-bit integer.
+: source keys
+:: An [=ordered set=] of [=strings=].
+: filters
+:: An [=attribution filter map=].
+: not_filters
```
Comment (apasel422) [inline index.bs:372]:
```suggestion
: <dfn>negated filters</dfn>
```

File: index.bs [hunk 2]
```diff
@@ -352,6 +352,38 @@ An attribution source is a [=struct=] with the following items:
 :: An [=attribution filter map=].
 : <dfn>debug key</dfn>
 :: Null or a non-negative 64-bit integer.
+: aggregatable source
```
Comment (apasel422) [inline index.bs:355]:
```suggestion
: <dfn>aggregatable source</dfn>
```

File: index.bs [hunk 3]
```diff
@@ -352,6 +352,38 @@ An attribution source is a [=struct=] with the following items:
...
+    non-negative 128-bit integers.
+
+</dl>
+
+<h3 dfn-type=dfn>Attribution aggregatable trigger data</h3>
+
+An attribution aggregatable trigger data is a [=struct=] with the following items:
+
+<dl dfn-for="attribution aggregatable trigger data">
+: key piece
```
Comment (apasel422) [inline index.bs:366]:
```suggestion
: <dfn>key piece</dfn>
```

File: index.bs [hunk 4]
```diff
@@ -352,6 +352,38 @@ An attribution source is a [=struct=] with the following items:
...
+:: A non-negative 128-bit integer.
+: source keys
```
Comment (apasel422) [inline index.bs:368]:
```suggestion
: <dfn>source keys</dfn>
```

File: index.bs [hunk 5]
```diff
@@ -352,6 +352,38 @@ An attribution source is a [=struct=] with the following items:
...
+:: An [=attribution filter map=].
+
+</dl>
+
+<h3 dfn-type=dfn>Attribution aggregatable trigger</h3>
+
+An attribution aggregatable trigger is a [=struct=] with the following items:
+
+<dl dfn-for="attribution aggregatable trigger">
+: trigger data
```
Comment (apasel422) [inline index.bs:382]:
```suggestion
: <dfn>trigger data</dfn>
```

File: index.bs [hunk 6]
```diff
@@ -352,6 +352,38 @@ An attribution source is a [=struct=] with the following items:
...
+:: A [=list=] of [=attribution aggregatable trigger data=].
+: values
```
Comment (apasel422) [inline index.bs:384]:
```suggestion
: <dfn>values</dfn>
```

File: index.bs [hunk 7]
```diff
@@ -378,6 +410,8 @@ An attribution trigger is a [=struct=] with the following items:
 :: An [=attribution filter map=].
 : <dfn>debug key</dfn>
 :: Null or a non-negative 64-bit integer.
+: aggregatable trigger
```
Comment (apasel422) [inline index.bs:413]:
```suggestion
: <dfn>aggregatable trigger</dfn>
```

File: index.bs [hunk 8]
```diff
@@ -417,6 +451,18 @@ An attribution report is a [=struct=] with the following items:
 
 </dl>
 
+<h3 dfn-type=dfn>Attribution aggregatable histogram</h3>
+
+An attribution aggregatable histogram is a [=struct=] with the following items:
+
+<dl dfn-for="">
```
Comment (apasel422) [inline index.bs:458]:
```suggestion
<dl dfn-for="attribution aggregatable histogram">
```

File: index.bs [hunk 9]
```diff
@@ -417,6 +451,18 @@ An attribution report is a [=struct=] with the following items:
...
+: key
```
Comment (apasel422) [inline index.bs:459]:
```suggestion
: <dfn>key</dfn>
```

File: index.bs [hunk 10]
```diff
@@ -417,6 +451,18 @@ An attribution report is a [=struct=] with the following items:
...
+:: A non-negative 128-bit integer.
+: value
```
Comment (apasel422) [inline index.bs:461]:
```suggestion
: <dfn>value</dfn>
```

**Review by apasel422 — DISMISSED** [review]

File: index.bs (hunk 2 above)
Comment (linnan-github) [reply index.bs:355]: Will add this when adding histogram creation algorithm.

File: index.bs (hunk 3 above)
Comment (linnan-github) [reply index.bs:366]: Will add this when adding histogram creation algorithm.

File: index.bs (hunk 4 above)
Comment (linnan-github) [reply index.bs:368]: Will add this when adding histogram creation algorithm.

File: index.bs (hunk 1 above)
Comment (linnan-github) [reply index.bs:372]: Will add this when adding histogram creation algorithm.

File: index.bs (hunk 5 above)
Comment (linnan-github) [reply index.bs:382]: Will add this when adding histogram creation algorithm.

File: index.bs (hunk 6 above)
Comment (linnan-github) [reply index.bs:384]: Will add this when adding histogram creation algorithm.

File: index.bs (hunk 7 above)
Comment (linnan-github) [reply index.bs:413]: Will add this when adding histogram creation algorithm.

File: index.bs (hunk 9 above)
Comment (linnan-github) [reply index.bs:459]: Will add this when adding histogram creation algorithm.

File: index.bs (hunk 10 above)
Comment (linnan-github) [reply index.bs:461]: Will add this when adding histogram creation algorithm.

**Pull request closed without merging**